apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDesignResources.md

```diff
@@ -31,7 +33,7 @@ The Add-ins design toolkit provides layouts for interface elements and commonly
 ### Fonts
 
 <ul class="md-list--flex">
-  <li class="mdut--half">Segoe UI and Fabric MDL2 icon font (coming soon)</li>
+  <li class="mdut--half">[Segoe UI and Fabric MDL2 icon font](https://microsoft.sharepoint.com/teams/OfficeUIFabric97/SitePages/Fabric%20MDL2%20Icons.aspx)</li>
```

Comment:
We need to put this info somewhere public--we can't refer external people to SharePoint pages

Reply:
Yeah, I don't yet have a public version of that, I'll ask Angela about it.

Reply:
aka.ms link will be great here as well in case it needs to be modified without a re-deploy

Reply:
@jordandrako this should just be removed from the public site altogether; internal resources will live on that site and shouldn't be here.